docs/api/commands/wait.mdx

```diff
@@ -60,18 +60,31 @@ Pass in an options object to change the default behavior of `cy.wait()`.
 | `requestTimeout`  | [`requestTimeout`](/guides/references/configuration#Timeouts)                                                                 | Overrides the global `requestTimeout` for this request. Defaults to `timeout`.           |
 | `responseTimeout` | [`responseTimeout`](/guides/references/configuration#Timeouts)                                                                | Overrides the global `responseTimeout` for this request. Defaults to `timeout`.          |
 
-### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
 
-#### When given a `time` argument:
+- `cy.wait()` ignores the subject it was given, except to pass it through if
+  given a `time` argument.
 
-- `cy.wait()` yields the same subject it was given.
-- It is [unsafe](/guides/core-concepts/retry-ability#Only-queries-are-retried)
-  to chain further commands that rely on the subject after `.wait()`.
+### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
 
-#### When given an `alias` argument:
+- `cy.wait()` has no implicit assertions.
 
-- `cy.wait()` 'yields an object containing the HTTP request and response
-  properties of the request.
+### Timeouts [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Timeouts)
+
+- When given a `time` argument, `cy.wait()` cannot time out.
+- When given an alias algument, `cy.wait()` can time out waiting for the request
+  to go out.
+- When given an alias algument, `cy.wait()` can time out waiting for the
+  response to return.
```

Comment:
```suggestion
  to go out or the
  response to return.
```

Reply:
I broke this out into two sentences because you can specify different request and response timeouts - it seemed like two different timeout options = two different ways it can time out.